cmd/add.go

```diff
@@ -51,7 +51,7 @@ changes to qri.`,
 	}
 
 	cmd.Flags().StringVarP(&o.File, "file", "f", "", "dataset data file in either yaml or json format")
-	cmd.Flags().StringVarP(&o.DataPath, "data", "d", "", "path to file or url to initialize from")
+	cmd.Flags().StringVarP(&o.BodyPath, "body", "b", "", "path to file or url to contents of dataset")
```

Comment:
This is very pedantic of me, but I parse this sentence like "path : file OR url : contents of dataset" when I think it's supposed to be "path : (file or url) : contents of dataset". Maybe change to "path to file or url for contents of dataset".